<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Terminal</class>
 <widget class="QWidget" name="Terminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #1e1e1e;
        color: #cccccc;
    }
    
    QTextEdit {
        background-color: #1e1e1e;
        color: #cccccc;
        border: none;
        font-family: &quot;Consolas&quot;, &quot;Monaco&quot;, &quot;Courier New&quot;, monospace;
        font-size: 12px;
        selection-background-color: #264f78;
    }
    
    QToolBar {
        background-color: #2d2d30;
        border-bottom: 1px solid #3e3e42;
        spacing: 5px;
        padding: 4px;
    }
    
    QPushButton {
        background-color: #3e3e42;
        border: 1px solid #6f6f6f;
        border-radius: 3px;
        color: #cccccc;
        padding: 4px 10px;
        font-size: 10px;
    }
    
    QPushButton:hover {
        background-color: #6f6f6f;
    }
    
    QPushButton:pressed {
        background-color: #2d2d30;
    }
    
    QLabel {
        color: #cccccc;
        font-size: 11px;
        padding: 0px 5px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="terminalToolbar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QFrame {
           background-color: #2d2d30;
           border-bottom: 1px solid #3e3e42;
       }
      </string>
     </property>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="terminalLabel">
        <property name="styleSheet">
         <string notr="true">
          QLabel {
              font-weight: bold;
              font-size: 11px;
              color: #98c379;
          }
         </string>
        </property>
        <property name="text">
         <string>⚡ TERMINAL</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear terminal</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none; padding: 0;</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Close terminal</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: transparent;
              border: none;
              font-size: 14px;
              font-weight: bold;
              padding: 0;
          }
          QPushButton:hover {
              background-color: #e06c75;
              color: white;
          }
         </string>
        </property>
        <property name="text">
         <string>×</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TerminalTextEdit" name="terminalOutput">
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TerminalTextEdit</class>
   <extends>QTextEdit</extends>
   <header>terminal.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
